types/pino/index.d.ts

```diff
@@ -17,11 +17,12 @@
 
 /// <reference types="node"/>
 
-import stream = require('stream');
-import http = require('http');
-import { EventEmitter } from 'events';
-import SonicBoom = require('sonic-boom');
-import * as pinoStdSerializers from 'pino-std-serializers';
+import stream = require("stream");
+import http = require("http");
+import { EventEmitter } from "events";
+import SonicBoom = require("sonic-boom");
+import * as pinoStdSerializers from "pino-std-serializers";
+import * as PinoPretty from "pino-pretty";
```

Comment:
`import { PrettyOptions } from "pino-pretty";`?

Reply:
This was my instinct too, but I couldn't figure out how to re-export it in the namespace when imported this way, since it would result in this, which doesn't work:
```typescript
type PrettyOptions = PrettyOptions;
```

Reply:
Makes sense.  I agree that it's not very important.  I mostly brought it up because, with this style of import, `PinoPretty` will not be callable, which might not be obvious to consumers.